content/en/apps/guides/messaging/gateways/gateway/troubleshooting.md

```diff
@@ -50,8 +50,36 @@ grep api/sms medic_medic_medic-api4.log
 ```
 If the above steps don't yield the problem, read the [Obtaining Logs]({{< ref "apps/guides/debugging/obtaining-logs#android-logs" >}}) page, note the make/model/android version their gateway handset is and escalate with those details to the PM/project techlead.
 
+**Verifying the SMS Gateway via a Test Message**
+
+We can verify whether the SMS Gateway app is responding by sending a test message via the web app. For example, let us try this in a deployed project.
+1. Login via the web app.
```

Comment:
`Log in` (verb) instead of `Login` (adjective).

Also, you might want to specify that you must log in as a user with administrative rights.

Reply:
Thanks. Done.